packages/ui/src/utils/stringUtils.ts

```diff
@@ -47,3 +47,23 @@ export const formatCountryAndState = (user: IUser): string => {
 
     return `${user.location_country}`;
 };
+
+export const removeAccents = (text: string): string => {
```

Comment:
Pas besoin de remplacer les accents sur le majuscules ? 
Si jamais y'a cette solution: https://stackoverflow.com/questions/990904/remove-accents-diacritics-in-a-string-in-javascript

Reply:
il y a un `toLowerCase` qui est àppelé, mais en effet je peux le transférer dans cette fontion